declarations/test.d.ts

```diff
@@ -0,0 +1,5 @@
+declare const enum TestConstants {
```

Comment:
>TestConstants [](start = 19, length = 13)

Can we keep this enum inside the test file and not add another file if all we are going to have is this enum? I think separate of declaration for test here might be an overkill.